public/src/actions/calls.jsx

```diff
@@ -54,6 +55,13 @@ export function setCallContactInfo(contactInfo) {
   };
 }
 
+export function setNoCallsAvailable(boolean) {
+  return {
+    type: SET_NO_CALLS_AVAILABLE,
+    payload: boolean
```

Comment:
Do we anticipate that after this is set to true it will be toggled to false?
I think we don't have a functionality to add contacts to a campaign's contact list, so this could just be an action with a type and the reducer can handle the logic of setting it to true.